app/presenters/notifications/setup/scheduled-notification-update-notify.presenter.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+/**
+ * Updates a scheduled notification to use the notify response data
+ * @module ScheduledNotificationsUpdateNotifyPresenter
```

Comment:
I am stuck on how to name this. 

This will be called after we have the notify response.